b2b-sample/guardio-insurance-sample-application/util/util/common/common.js

```diff
@@ -73,6 +73,24 @@ function copyTheTextToClipboard(text, toaster) {
     infoTypeDialog(toaster, "Text copied to clipboard");
 }
 
+/**
+ * 
+ * @returns random generatored rgb colour
+ */
+function random_rgba() {
+    var o = Math.round, r = Math.random, s = 255;
+    return 'rgba(' + o(r()*s) + ',' + o(r()*s) + ',' + o(r()*s) + ',' + r().toFixed(1) + ')';
+}
+
+/**
+ * operations that we can do on PATCH methods
```

Comment:
`Operations that we can do on PATCH methods.` Formatting issues check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b